CMakeLists.txt

```diff
@@ -4,6 +4,8 @@ set(CMAKE_CXX_STANDARD 20)
 
 project(d2tm)
 
+SET(OpenGL_GL_PREFERENCE "GLVND")
```

Comment:
We can probably fix this in the Allegro library.
When we switch to SDL2, this is probably no longer needed.